.github/workflows/openvino_code.yml

```diff
@@ -0,0 +1,65 @@
+name: OpenVINO Code Extension Workflow
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      - 'modules/openvino_code/**'
+      - '.github/workflows/openvino_code.yml'
+
+concurrency:
+  group: ${{ github.head_ref || github.ref_name }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    working-directory: ./modules/openvino_code
+
+jobs:
+  check_extension:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
```

Comment:
I suppose these explicit `ref` and `repository` are not required

Reply:
Fixed